.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,49 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if command is '/request_review'
+        id: check-command
+        run: |
+          if [[ $(echo "${{ github.event.comment.body }}" | tr '[:upper:]' '[:lower:]') == "/request_review"* ]]; then
+            echo "::set-output name=command::true"
```

Comment:
Lets use `GITHUB_OUTPUT` from now on: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/